instrumentation/spring/spring-kafka-2.7/library/src/main/java/io/opentelemetry/instrumentation/spring/kafka/v2_7/InstrumentedRecordInterceptor.java

```diff
@@ -31,11 +50,26 @@
     this.decorated = decorated;
   }
 
-  @SuppressWarnings("deprecation") // implementing deprecated method for better compatibility
-  @Override
+  @SuppressWarnings({
+    "deprecation",
+    "unchecked"
+  }) // implementing deprecated method for better compatibility
+  // deprecated method removed in 3.0
+  // @Override
```

Comment:
I think you could keep the `@Override` as we are always compiling against the early version, or am I missing something?

Reply:
The test latest deps task will try to compile this against the latest version.

Reply:
Looks like it's working just fine with the annotation; I must have messed something up when running this locally.
Thanks for 🦅 👁️